### DISCUSSION TITLE
403

### DISCUSSION THREAD

**Discussion by nephen** [linweiyuan/go-chatgpt-api#115] [Ideas]
![image](https://user-images.githubusercontent.com/7166946/236753595-75d9ced7-090f-49c3-bbd6-701b94967216.png)
虽然403了，但是还是可以获取到__cf_bm的，感觉可以定期请求获取这个cookie，后面就可以正常使用了。

**Comment by linweiyuan** [comment]
本来就是这样做的，但是昨天有人攻击我服务器，我删了

今天接下 Cloudflare 试下免费的 DDoS 防护好不好使，没问题再重新提供 API

**Reply by nephen** [reply]
意思是直接在go-chatgpt-api里面定期get https://chat.openai.com/api/auth/session就行了，不管它是不是200状态码，感觉是可以的，我测试下

**Reply by linweiyuan** [reply]
试过了，除非不触发验证码，需要验证码的拿到也不合法

长度 209 的不一定合法，长度 145 的一定不合法
